docs/source/author/writing_guide.rst

```diff
@@ -72,6 +57,9 @@ Blockquotes
 
   > Oh, you can *put* **Markdown** into a blockquote.
 
+.. raw:: html
+  :file: ../_static/html_snippets/markdown_example_blockquote.html
```

Comment:
Might want to check that the new line continuing a quote thing does actually put in a new line, or if it follows the same rules we went over last week - continuing on the same line